silk/request_filters.py

```diff
@@ -163,7 +164,15 @@ def __str__(self):
         return '#queries >= %s' % self.value
 
     def contribute_to_query_set(self, query_set):
-        return query_set.annotate(num_queries=Count('queries'))
+        return query_set.annotate(
+            # This is overly complex due to Oracle not accepting group by on TextField
```

Comment:
Is `contribute_to_query_set` used anywhere in the hot path (for silk to ingest data)?  If it is, this change may incur a significant performance degradation to host django apps.